bats/tests/helpers/utils.bats

```diff
@@ -1,5 +1,8 @@
 load '../helpers/load'
 
+# Don't display the info.bash output if this is the only test running
+RD_INFO=false
```

Comment:
Should this be in `local_setup_file` instead? (I think having it bare sometimes leaks into other tests? Check with `bats … tests/helpers/utils.bats tests/something/else.bats`)

Reply:
Jan brings up that this wouldn't help: we only print the info in the first test, so we still wouldn't print anything even if the second test doesn't suppress it.  So as long as having `utils.bats` as one of the _later_ tests doesn't suppress the info banner, the code as-is is good enough.

Reply:
I made it so you can still force it to always show the info by explicitly setting `RD_INFO=1`, but yes, I think this works as expected in all practical cases.